rele/client.py

```diff
@@ -42,9 +42,15 @@ def __init__(self, gc_project_id, credentials, default_ack_deadline=None):
         self._ack_deadline = default_ack_deadline or DEFAULT_ACK_DEADLINE
 
         if USE_EMULATOR:
-            self._client = pubsub_v1.SubscriberClient()
+            self._subscriber_client = pubsub_v1.SubscriberClient()
+            self._publisher_subscriber_client = pubsub_v1.PublisherClient()
         else:
-            self._client = pubsub_v1.SubscriberClient(credentials=credentials)
+            self._subscriber_client = pubsub_v1.SubscriberClient(
```

Comment:
I wouldn't touch `self._client`, it's the main private attribute of the `Subscriber` class, it doesn't deserve to become ugly just because we need to cover a corner case :grin: 

In addition to that, we should probably use the global publisher anyway, so no need to initialize a publishing client at all

Reply:
I think `self._subscriber_client` is more descriptive. Anyway, `Subscriber` class should not have dependencies with the `Publisher` class.